lib/super_tooltip.dart

```diff
@@ -87,7 +87,6 @@ class SuperTooltip extends StatefulWidget {
     this.touchThroughAreaShape = ClipAreaShape.oval,
     this.touchThroughAreaCornerRadius = 5.0,
     this.touchThrougArea,
-    // FIXME: Why does a 0.0 borderwidth still show a border?
```

Comment:
Have you tried this? does a borderWith 0 still show a border?

Reply:
Yes I tried, it doesn't